### PR TITLE
[115] Create Freak model and add Hard-coded data

### PR DESCRIPTION
**Pull request by salcudeann**
## Description

In this PR we've created a Freak model and other models needed  for this type.
Also, we've created a json file which contain mock data for Freak model. This data helped us to execute a GraphQL query and check if everything works fine.

## Related

[[115] Get hardcoded Freaks](https://trello.com/c/en8qlw3c/115-get-hardcoded-freaks)

## Steps to Test or Reproduce

1. Create  a JSON file and populate with mock data
``` JSON
[
  {
    "id":"1",
    "name":"Doru Pirvu",
    "photo":{
      "id":"1",
      "uri":"https://bit.ly/3y7afHO"
    },
    "description":"The boss.",
    "skills":[
      {
        "id":"1",
        "name":"Project manangement"
      },
      {
        "id":"2",
        "name":"AWS Architect"
      }
    ],
    "projects":[
      {
        "id":"1",
        "name":"EPIX",
        "description":"An American premium cable and satellite TV network.",
        "photo":{
          "id":"1",
          "uri":"https: //bit.ly/3y7afHO"
        },
        "technologies":[
          {
            "id":"2",
            "name":"Ruby",
            "description":"A popular backend language."
          }
        ]
      }
    ],
    "role":{
      "id":"1",
      "name":"Founder"
    },
    "level":{
      "id":"3",
      "name":"Advanced"
    },
    "norm":{
      "id":"1",
      "name":"full-time"
    },
    "speciality":[
      {
        "id":"2",
        "name":"Backend"
      }
    ]
  },
  {
    "id":"2",
    "name":"Alex Negru",
    "photo":{
      "id":"1",
      "uri":"https://bit.ly/3y7afHO"
    },
    "description":"Intern driver.",
    "skills":[
      {
        "id":"1",
        "name":"Backend developer"
      },
      {
        "id":"2",
        "name":"driver"
      }
    ],
    "projects":[
      {
        "id":"1",
        "name":"EPIX",
        "description":"An American premium cable and satellite TV network.",
        "photo":{
          "id":"1",
          "uri":"https://bit.ly/3rB9Yu9"
        },
        "technologies":[
          {
            "id":"2",
            "name":"Ruby",
            "description":"A popular backend language."
          }
        ]
      }
    ],
    "role":{
      "id":"1",
      "name":"Software developer"
    },
    "level":{
      "id":"3",
      "name":"Advanced"
    },
    "norm":{
      "id":"1",
      "name":"full-time"
    },
    "speciality":[
      {
        "id":"2",
        "name":"Backend"
      }
    ]
  }
]
```
2. Write a GraphQL query to get data 
``` GraphQl
query GetFreaks {
	freaks {
		id
		name
		photo {
			id
			uri
		}
		description
		skills {
			id
			name
		}
		projects {
			id
			name
			description
			photo {
				id
				uri
			}
			technologies {
				id
				name
				description
			}
		}
		role {
			id
			name
		}
		level {
			id
			name
		}
		norm {
			id
			name
		}
		speciality {
			id
			name
		}
	}
}
```
3. Execute this query using a POST request to _http://localhost:3000/graphql_ using Insomnia

### CODE REVIEW

**Review by alex-damian-negru — DISMISSED** [review]
How can I test this?

Please add a PR description from now on.

**Comment by alex-damian-negru** [comment]
[Get hardcoded Freaks](https://trello.com/c/en8qlw3c/115-get-hardcoded-freaks)

**Review by pirvudoru — APPROVED** [review]

File: app/graphql/models/level.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module Models
```
Comment (pirvudoru) [inline app/graphql/models/level.rb:3]: This module should be named "Types"

File: tatus [hunk 2]
```diff
@@ -0,0 +1,29 @@
+[33mcommit 758be0ccab28ce82cda6593a763e253cf3c6999d[m
```
Comment (pirvudoru) [inline tatus:1]: What's this file about ? :) I assume we can delete it.

File: app/graphql/models/level.rb (hunk 1 above)
Comment (pirvudoru) [reply app/graphql/models/level.rb:3]: Here you got it right: 
https://github.com/Agilefreaks/FreaksCatalog-API/pull/7/files#diff-eb9a313cd462df6ec80d88d35d2eae1fddcd38136099d37d431738e500a350e0R4

All other Models:: should follow the same pattern. See how you are inheriting from `Types::BaseObject`

**Review by pirvudoru — CHANGES_REQUESTED** [review]

**Review by pirvudoru — APPROVED** [review]
:100:

**Pull request merged**